src/progress/pct/compile.p

```diff
@@ -281,7 +283,7 @@ PROCEDURE compileXref.
   IF (opError) THEN RETURN.
   ASSIGN opError = NOT createDir(PCTDir, cBase).
   IF (opError) THEN RETURN.
-  cSaveDir = IF cFileExt = ".cls" OR lRelative THEN outputDir ELSE outputDir + '/':U + cBase.
+  cSaveDir = IF DestDir EQ ? THEN ? ELSE (IF cFileExt = ".cls" OR lRelative THEN outputDir ELSE outputDir + '/':U + cBase).
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Line is 123 characters long, exceeding maximum value of 120 [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=rssw-oe-main%3Aeu.rssw.antlr.proparse.checks.LinesTooLong)